src/sentry/static/sentry/app/icons/icon-calendar.svg

```diff
@@ -1,11 +0,0 @@
-<svg
```

Comment:
Can we make sure this isn't used in getsentry

Reply:
Yep - this was not in getsentry